pool/web/middleware/middleware_tenant.ml

```diff
@@ -1,27 +1,41 @@
+let tenant_of_request req =
+  let open Utils.Lwt_result.Infix in
+  (* TODO handle PREFIX_PATH of Tenant URLs, multiple tenants behind the same
```

Comment:
Do we have an issue for that? Should we create one if not?

Reply:
I created one for it. Thanks for pointing it out.